daemon/api_test.go

```diff
@@ -2402,7 +2402,9 @@ func (s *apiSuite) TestGetEvents(c *check.C) {
 
 func setupChanges(st *state.State) []string {
 	chg1 := st.NewChange("install", "install...")
-	chg1.Set("snap-names", []string{"funky-snap-name"})
+	chg1.Set("api-data", map[string]interface{}{
+		"snap-names": []string{"funky-snap-name"},
```

Comment:
See above (I added api-data here and with that I wrapped it, I don't particularly mind but it seems easier to read this way now that there is one more level in the data structure.)